grid2op/Backend/backend.py

```diff
@@ -179,6 +179,8 @@ def __init__(self,
         #: There is a difference between this and the class attribute.
         #: You should not worry about the class attribute of the backend in :func:`Backend.apply_action`
         self.n_busbar_per_sub: int = DEFAULT_N_BUSBAR_PER_SUB
+
+        self.set_shedding(allow_shedding)
```

Comment:
Instead of using this kind of method to tell the backend to support "shedding" (that I would not call shedding, but rather "disconnecting" or something else - i'm not a fan of "disconnecting" either) can you do the same thing as I did in the `support_multiple_busbar_per_substation` 

That is : having a default attribute (in my case `_missing_two_busbars_support_info` so in your case `_missing_disconnecting_info`) that the creator of the backend can either activate in its backend (by calling in my case `can_handle_more_than_2_busbar`) or deactivate (by calling explicitly `cannot_handle_more_than_2_busbar` ) a call to any of these functions will toggle to `False` the `_missing_two_busbars_support_info` flag and have another attribute (in my example `n_busbar_per_sub` in your example the `allow_shedding` flag, which in my opinion should be private)

Reply:
Have changed the interface for this now to match the missing busbars approach. Since n_busbars is not private, should I not keep that the same too if I make the _missing_detachment_support private?